nodejs/week1/circle.js

```diff
@@ -0,0 +1,21 @@
+class circle {
+          constructor(radius) {
+                    this.radius = radius;
+          }
+
+          getDiameter() {
+                    return 2 * this.radius;
+          }
+          getArea() {
+                    return Math.PI * this.radius * this.radius;
+          }
+          getCircumference() {
+                    return 2 * Math.PI * this.radius;
+          }
+}
+
+const c1 = new circle(10);
```

Comment:
The name of variables also should be self-descriptive. So it would be great if you rename it to circle1 = new Circle(10)